src/index.js

```diff
@@ -9,7 +9,7 @@ const leaderName = document.querySelector('#name');
 const leaderScore = document.querySelector('#number');
 const form = document.querySelector('#form');
 
-const myGameURL = 'ABTobGSw40a3MfxWImKz';
+const myGameURL = 'ABSobGSw40a3MfxWImKz';
 export const baseURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${myGameURL}/scores/`;
 
 form.addEventListener('submit', (e) => {
```

Comment:
- [ ] **[REQUIRED]** Kindly ensure that your app is properly styled and its layout looks like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). 

Your Implementation | Wireframe
:---:|:---:
![](https://i.imgur.com/csvVEll.png) | ![](https://i.imgur.com/DHVqMA6.png)


- [ ] **[OPTIONAL]** I was not able to see my score after adding a score, and clicked refresh button. No scores is displayed. I suggest you fix this bug so that your project works as expected.